publication-commons/src/test/java/no/unit/nva/publication/service/impl/DynamoDBPublicationServiceTest.java

```diff
@@ -460,6 +461,26 @@ public void createPublicationReturnsPublicationWithIdentifierWhenInputIsValid()
         assertThat(actual.getIdentifier(), is(not(nullValue())));
     }
 
+    @Test
+    public void deletePublicationCanMarkDraftForDeletion() throws ApiGatewayException {
+        Publication publication = publicationWithIdentifier();
+        publicationService.createPublication(publication);
+
+        publicationService.deletePublication(publication.getIdentifier());
```

Comment:
I am wondering, since there will be an actual deletion, should we call this function something else?